app/controllers/fast_controller.rb

```diff
@@ -20,55 +20,40 @@ def show
   private
 
   def lookups(query)
-    # Try first without wildcard.
```

Comment:
As far as I can tell from https://www.oclc.org/developer/api/oclc-apis/fast-api/assign-fast.en.html, there is no support for wildcard queries anymore.